From b47f952a8b6823e40da84dac916abe34e416e705 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 25 Jul 2018 18:26:53 +0000
Subject: [PATCH] MANTA-3753 muppet should leverage autometadata instead of
 checking nictags (bump version) Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index da15204..3c0f21b 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "muppet",
     "description": "Joyent's Load Balancer",
-    "version": "1.0.1",
+    "version": "1.1.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
-- 
2.21.0

